src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -324,8 +324,9 @@ public function getData()
         $data = array();
 
         // Required request parameters for Apple Session object.
-        $data['merchantIdentifier'] = $this->getMerchantIdentifier();
+        $data['validationUrl'] = $this->getValidationUrl();
```

Comment:
is this related to the cert at all? if not, could you make this change a separate PR? not clear what this is for